website/addons/osfstorage/settings/defaults.py

```diff
@@ -27,14 +28,14 @@
 # }
 
 WATERBUTLER_CREDENTIALS = {
-    'username': 'changeme',
-    'token': 'changeme',
-    'region': 'changeme'
+    'storage': {}
 }
 
 WATERBUTLER_SETTINGS = {
-    'provider': 'cloudfiles',
-    'container': 'changeme',
+    'storage': {
+        'provider': 'filesystem',
+        'folder': os.path.join(os.getcwd(), 'osfstoragecache'),
```

Comment:
`os.getcwd()` may vary depending on where the server is run from. Better to use a constant from `website.settings`, e.g. `BASE_PATH`.